bundles/test/org.eclipse.smarthome.test/src/main/java/org/eclipse/smarthome/test/java/JavaOSGiTest.java

```diff
@@ -58,7 +58,7 @@ public void bindBundleContext() {
      *
      * @return bundle context
      */
-    private BundleContext getBundleContext() {
+    protected BundleContext getBundleContext() {
```

Comment:
This method is called by `bindBundleContext()` that is annotated with `@Before`.

You should access the bundleContext member directly -- it is already `protected`.
Perhaps this method name is confusing and should be named `initBundleContext`.

Reply:
+1 - would you mind changing the method name accordingly while you are at it anyway?

Reply:
agreed.